common/azure-common/src/main/java/com/azure/common/implementation/SwaggerMethodParser.java

```diff
@@ -526,4 +519,31 @@ private String applySubstitutions(String originalValue, Iterable<Substitution> s
 
         return result;
     }
+
+    private HashMap<Integer, UnexpectedException> processUnexpectedResponseExceptionTypes() {
```

Comment:
Return the `Map` interface, not a concrete implementation

Reply:
Changed to map